sentence_transformers/sparse_encoder/SparseEncoder.py

```diff
@@ -1022,6 +999,34 @@ def max_seq_length(self, value) -> None:
         """
         self._first_module().max_seq_length = value
 
+    @property
+    def chunk_size(self) -> int | None:
```

Comment:
Maybe it should be the `splade_pooling_chunk_size` or something? And it's not very clear what this does right now